### PR TITLE
add echo example with automatic parsing of function id, hash, cape token

### PR DESCRIPTION
**Pull request by ekloberdanz**
Adds a script that automatically parses Cape function id, function hash, and token to deploy and run the echo example. Next PR will update the ReadMe to reflect that the user can just run python3 deploy_run_echo.py without needing to set environment variables for function id etc.

### CODE REVIEW

File: examples/deploy_run_echo.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    elif "Function Hash" in i:
+        function_hash = i.split(" ")
+        CAPE_FUNCTION_HASH = function_hash[3]
+
+# Parse cape/auth for CAPE_TOKEN
+HOME = os.getenv("HOME")
+auth_file = f"{HOME}/.config/cape/auth"
+f = open(auth_file)
+cape_auth = json.load(f)
+CAPE_TOKEN = cape_auth["access_token"]
```
Comment (jvmncs) [inline examples/deploy_run_echo.py]: this looks like it duplicates the default behavior in the `Cape` client, have a look [here](https://github.com/capeprivacy/pycape/blob/efdadc389b53b7199354d2a46b0d94acca70acc6/pycape/cape.py#L68-L70)

Comment (ekloberdanz) [reply examples/deploy_run_echo.py]: I removed the redundant token parsing, will amend the readme in pycape/examples to reflect that CAPE_TOKEN does not need to be set as an environment variable.

File: examples/deploy_run_echo.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import subprocess
+
+from pycape import Cape
+from pycape import FunctionRef
+
+CAPE_HOST = "wss://hackathon.capeprivacy.com"
```
Comment (jvmncs) [inline examples/deploy_run_echo.py]:
```suggestion
CAPE_HOST = "wss://enclave.capeprivacy.com"
```

**Review by jvmncs — APPROVED** [review]
lgtm w/ comment

**Comment by jvmncs** [comment]
@ekloberdanz looks like these commits haven't been signed, which is blocking the PR from being merged. you will want to set up [GPG signing for git](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification) if you haven't already. once you've done that, you can rebase this branch against main w/ something like this: 
```
git rebase origin/main -x 'git commit --no-edit --amend -n -S'
```

**Comment by jvmncs** [comment]
superceded by #57

**Pull request closed without merging**